### PR TITLE
httpd package

### PR DESCRIPTION
**Pull request by Abhijeet-Jadhav**

### CODE REVIEW

File: repo/packages/H/httpd/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+ "images": {
+   "icon-large": "http://2.bp.blogspot.com/-1pgWIqTq1b0/VqhcarY1BOI/AAAAAAAAA6I/4_L9ZSCRxIk/s400/apache318x2601.png",
```
Comment (BenWhitehead) [inline repo/packages/H/httpd/0/resource.json]: this should be `icon-small`. Right now there are two `icon-large`.

**Comment by Abhijeet-Jadhav** [comment]
Missed to correct that icon-small. Thanks @BenWhitehead .

**Comment by ryadav88** [comment]
@Abhijeet-Jadhav Are you still working on this package? Looks like it needs to be updated to add various fields in config json.

**Comment by Abhijeet-Jadhav** [comment]
@ryadav88 Okay, do we have minimal requirements on the information needed for each package file?

**Comment by ryadav88** [comment]
@Abhijeet-Jadhav it would depend on the service. A good place to start would be the Universe README and a recently merged package like this one: `https://github.com/mesosphere/universe/tree/version-3.x/repo/packages/G/grafana/0`

**Comment by ryadav88** [comment]
@Abhijeet-Jadhav any update here?

**Comment by Abhijeet-Jadhav** [comment]
@ryadav88 Testing part is left. Will finish up by this weekend.

**Comment by ryadav88** [comment]
@Abhijeet-Jadhav circling back. any updates here?

**Comment by Abhijeet-Jadhav** [comment]
Problem running the universe-server locally for trying to the run the package. Nothing on stderr and stdout. Tried debugging have to recreate and test again.

**Comment by ryadav88** [comment]
Closing PR because of inactivity. Feel to reopen at a later time.

**Pull request closed without merging**